src/Pages/AddUser/index.tsx

```diff
@@ -34,41 +55,38 @@ export default function InputFormProps() {
     mode: 'onBlur',
     resolver: zodResolver(schema),
   });
-  const onSubmit = (data: FormData) => {
+  const onSubmit = (data: FormData, e: any) => {
+    e.preventDefault();
+    e.target.reset();
+    setOpen(true);
     mutate(data);
   };
+  
   return (
-    <Layout>
-      <Box
-        sx={{
-          py: 2,
-          px: 40,
-          display: 'grid',
-          gap: 2,
-          alignItems: 'center',
-        }}
-      >
-      <form onSubmit={handleSubmit(onSubmit)}>
-        <Stack spacing={1}>
-          {["firstName", "surName", "number", "email", "photoUrl"].map((i, index) => (
-            <div key={index}>
-              <Input 
-                sx={{
-                  width: '60vw',
-                }}
-                {...register(i as keyof TFieldValues, { required: true })}
-                placeholder={`Enter Your ${i}*`}  
-              />
-              {errors[i as keyof TFieldValues] && <p className={style.error}>{i} is required</p>}
-            </div>
-          ))}
-          <Button type="submit">Submit</Button>
-        </Stack>
-      </form>
-      {isLoading && <p>Adding user ...</p>}
-      {isError && error instanceof Error && <p>An error occurred: {error.message}</p>}
-      {isSuccess && <p>User added successfully!</p>}
-      </Box>
-    </Layout>
+      <Main open={openSidebar} sx={styles.myBox}>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <Stack spacing={1}>
+            {listInput.map((text, index) => (
+              <div key={index}>
+                <Input 
+                  sx={styles.myInput}
+                  {...register(text as keyof TFieldValues, { required: true })}
+                  placeholder={`Enter Your ${text}*`}  
+                />
+                {errors[text as keyof TFieldValues] && <Typography variant="caption" display="block" gutterBottom color='error'>{text} is required</Typography>}
+              </div>
+            ))}
+            <Button type="submit">Submit</Button>
+          </Stack>
+        </form>
+        {isError && error instanceof Error && <p>An error occurred: {error.message}</p>}
+        {isSuccess && 
```

Comment:
use notistack package instead of mui snackbar

Reply:
I used notistack on the Signin page
that I have used every mode in the project